tools/slack/bot.py

```diff
@@ -0,0 +1,31 @@
+import os
+from slack_sdk import WebClient
+from pathlib import Path
+from dotenv import load_dotenv
+from flask import Flask
+from slackeventsapi import SlackEventAdapter
+
+#Create .env file and load it
+load_dotenv()
+app = Flask(__name__)
```

Comment:
Add in a doc ways to have the webserver with pros/cons?
Flask, Hue url, Hue standalone command?

Reply:
I am not able to get what is meant by "Hue url", so like we can have something like "demo.gethue.com/slack/events" or "demo.gethue.com/slack/actions" ?

And "Hue standalone command" ? More like a "Slash command" in slack ?

Reply:
The webserver could be: Flask, directly part of Hue, running outside of Hue. 
I put an example of Hue/Django command showing how from the Hue code base a server can be started on the side.